Tests/Functional/DatabaseControllerTest.php

```diff
@@ -1,4 +1,6 @@
 <?php
+namespace Aoe\t3deploy\Tests\Functional;
```

Comment:
I would opt for UpperCamelCase in the namespace, thus Aoe\T3deploy\Tests\Functional.

Even though not being explicitly stated in PSR-4, this feels like an implicit convention to me, at least in the TYPO3 context.

Reply:
I realised this later. It has been fixed in "feature-implement-namespace" branch.